ZenPacks/zenoss/ZenPackLib/tests/test_impact_triggers.py

```diff
@@ -77,23 +67,30 @@ class ImpactTestCase(object):
     base: [SubComponent]
     monitoring_templates: [AuxComponent]
     impacted_by: [basicDevice, subComponents]
+device_classes:
+  /Server/Basic:
+    remove: true
+    zProperties:
+      zPythonClass: ZenPacks.zenoss.MockZenPack.BasicDevice
+
 """
 
 
-class TestImpactTriggers(ImpactTestCase):
+class TestImpactTriggers(ImpactTestCase, ZPLTestMockZenPack):
     """Test Impact Triggers"""
 
+    datasources = []
```

Comment:
Class level properties should not be set to mutable values like this. It is a common source of bugs. This goes for thresholds on the following line too. It also goes for the same properties on the ZPLTestMockZenPack class.

The same is also true for method/function default values in `MockZenPack.__init__(self, dmd, name, datasources=[], thresholds=[])`.